obspy/core/tests/test_utcdatetime.py

```diff
@@ -31,7 +31,9 @@ def test_from_string(self):
         self.assertEqual(dt, UTCDateTime(2009, 7, 1, 12, 12, 12))
         dt = UTCDateTime("19700101")
         self.assertEqual(dt, UTCDateTime(1970, 1, 1, 0, 0))
-        # non ISO8601 strings should raise an exception
+        dt = UTCDateTime("1970/01/01 12:23:34")
```

Comment:
I think 1970/01/01 should in general not be used for these tests (at least not solely) because in principle it could mask severe problems parsing the whole date (as it is the 0-date kind of)